dashboard/src/components/TimeRange/QuickSelectSection.tsx

```diff
@@ -28,15 +31,21 @@ export function QuickSelectSection({ selectedRange, onRangeSelect }: QuickSelect
   const renderButton = (value: Exclude<TimeRangeValue, 'custom'>) => {
     const preset = byValue.get(value);
     if (!preset) return null;
+    const isAllowed = allowedValues ? allowedValues.includes(value) : true;
```

Comment:
```suggestion
    const isAllowed = !allowedValues || allowedValues.includes(value);
```

intention is a bit clearer if you write it like this